action-processor/create-action.js

```diff
@@ -489,13 +489,15 @@ async function createSecureMessage (featureName, action){
     let savedUser = await localStorage.getItem(featureName, action.data.FromUser)
     let response = savedUser.response
     
-    let responseData = await messageService.createDraft(response.Id)
-    let draftId = responseData.Id
+    let createDraftResponse = await messageService.createDraft(response.Id)
+    let draftId = createDraftResponse.Id
 
     var sendData = {}
+    sendData.FromId = response.Id // Only used to populate header, should really be a param of sendMessage
+
     sendData.Subject = action.data.Subject
-    sendData.Message = action.data.Message
-    sendData.FromId = response.Id
```

Comment:
you don't need FromId  to createDraft anymore?

Reply:
Moved above, with comment; shouldn't be in payload as actually only used in header